Froggie/Test/Froggie.Api.Test/Integration/Groups/CreateGroupTest.cs

```diff
@@ -10,15 +12,22 @@ public sealed class CreateGroupTest : ApiIntegrationTest<CreateGroupController>
     [Test]
     public async ValueTask CreateTask_Success()

```

Comment:
Why is this called `CreateTask_Success()`? Seems like it should maybe be called `CreateGroup_Success()`?